server/.env

```diff
@@ -1,2 +1,2 @@
-MONGO_STR=<database_string>
+MONGO_STR=mongodb://admin.localhost:27017/walk2win-test

```

Comment:
Can you please keep mongo connection string as `<database_string>` because dev will use what they want :)

Reply:
sure I will update it. It was a mistake.

Reply:
Updated. :+1: